meta-aos-vm-common/recipes-aos/aos-clearhsm/aos-clearhsm.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Aos script to clear PKCS11 storage"
```

Comment:
should we move this receipt to meta-aos? I mean this functionality be used in another target not only for VM?

Reply:
I guess it will be difficult because aos-clearhsm.sh will be different.

Reply:
It is platform specific. Should be in meta-aos-vm.